src/components/graph/graph.builder.js

```diff
@@ -143,6 +146,9 @@ function buildLinkProps(link, nodes, links, config, linkCallbacks, highlightedNo
         source,
         stroke,
         strokeWidth,
+        strokeDasharray,
```

Comment:
These changes look good, but without adding the new configs to the [graph.config](https://github.com/danielcaldas/react-d3-graph/blob/eb1c81dfeb3835bb87d10dd570ecbec236f53157/src/components/graph/graph.config.js#L240) we won't be able to use them.

Please make sure you change the [graph.config.js](https://github.com/danielcaldas/react-d3-graph/blob/eb1c81dfeb3835bb87d10dd570ecbec236f53157/src/components/graph/graph.config.js#L240) with these new properties and add the proper default values for each one of them.